Dockerfile

```diff
@@ -12,7 +12,6 @@ FROM rclone/rclone:1.59.0 AS image-linux-armv7
 
 # for linux/arm/v6 platform
 FROM alpine:3.15 AS image-linux-armv6
-
```

Comment:
Don't delete blank lines, keep them to make it easier to read.

Reply:
I put it together because those two lines are needed for ARMv6 platform.